packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -91,8 +96,8 @@ const IModelLoader = React.memo(
       );
       let imodelConnection: IModelConnection | undefined;
       // create a new imodelConnection for the passed project and imodel ids or local file
-      if (blankConnection) {
-        imodelConnection = BlankConnection.create(blankConnection);
+      if (iTwinId && !iModelId) {
```

Comment:
should this include the blankConnection as well? Or should we try and remove the need for the blankConnection as well?

Reply:
Considered it, but decided to play it safe and allow an override for now.

Reply:
Could extract the overrides to separate/different props...

Reply:
actually doesnt really make sense for check the blankConnection because its optional now